README.md

```diff
@@ -764,7 +777,7 @@ If you're opening issues related to these, please mention the version that the i
 License
 -------
 
-Pa11y is licensed under the [Lesser General Public License (LGPL-3.0)][info-license].  
+Pa11y is licensed under the [Lesser General Public License (LGPL-3.0)][info-license].
```

Comment:
👍  Have gone with `<br>` (sans forward slash) based on [Stack Overflow](http://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br), unless you have a preference for how you've given i?